### PR TITLE
Release 3.0

### PR DESCRIPTION
**Pull request by Scooletz**
This PR bumps up `NServiceBus.Metrics.ServiceControl`  to version 3.0 which is dependent on:
- NServiceBus.Metrics 3.0
- NServiceBus 7.0

Additionally, it builds the package with a new csproj build system, targeting both: `netstandard2.0` and `net452`.

### CODE REVIEW

**Comment by Scooletz** [comment]
I got a binding redirect issue in tests with this. Any help greatly appreciated @Particular/metrics-maintainers .

**Comment by seanfarmar** [comment]
@Scooletz happy to help if I can

**Comment by seanfarmar** [comment]
@Scooletz the build seems to be broken?

**Comment by Scooletz** [comment]
@seanfarmar The build is broken by a single test that I fail to reproduce on my machine. That is because of the redirect issue I have. If you could run a build locally and test whether you got a redirect issue (my case) or only one test failing (TC case), that would bring us closer to the solution.
Also, after @bording PRs this branch will shortly receive a new package of SC.Monitoring.Data.

**Comment by Scooletz** [comment]
The PR this PR was waiting for has been merged https://github.com/Particular/ServiceControl.Monitoring.Data/pull/9 Once, the package is published we can move forward with it.

**Comment by Scooletz** [comment]
Ping @seanfarmar Have you tried to reproduce it locally?

**Comment by seanfarmar** [comment]
I'll check again

**Comment by Scooletz** [comment]
Ping @bording

**Comment by bording** [comment]
LGTM

BTW, looks like the merge conflict is because there was a change made to master that was never merged back to develop.

File: GitVersion.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ﻿assembly-versioning-scheme: Major
-next-version: 1.0.0
+next-version: 2.0.0
```
Comment (bording) [inline GitVersion.yml]: Seems like develop should have already been on 3.0 here?

**Comment by Scooletz** [comment]
I know about the conflict. Let me resolve it right now with commit-tree, but this might reappear when we release '2.0'

**Comment by Scooletz** [comment]
Closing this as it brings no value (just a dangling PR that will be needed once we release a stable core)

**Pull request closed without merging**